charts/cass-operator-chart/Chart.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: v2
```

Comment:
I would be the helm v2 install base out there is pretty wide. Does using this api version preclude us from supporting those folks? https://helm.sh/docs/topics/charts/#the-apiversion-field

Reply:
I thought that we were intentionally targeting only Helm v2.  From what I understand, Helm v2 is a can of worms.

@jimdickinson ?

Reply:
For what it's worth, among the people at KubeCon last year there were quite a few people that would have been quite happy to see Helm v2 burned to the ground, and pretty much everyone was glad to see Helm v3 get released. I don't know the technical details for all the angst with respect to v2, but I think the community has moved on.

Reply:
I thought the v3 binary was totally compatible with v2 charts?

Reply:
I will go and take a look, but if we intentionally release a chart that is compatible with Helm 2, then we are kind of saying that we support Helm 2.  Do we want to debug issues people might hit with Helm 2?

Reply:
Helm 3 does support Helm api v1 ( that is, Helm 2 ) charts.  The three main differences I see:

1. instead of requirements.yaml there is Chart.yaml in Helm 3
2. Helm 2 charts do not support "library charts", which are useful for sharing values between multiple charts to keep things DRY.
3. Putting out a Helm 2 chart sends a message that we support Helm 2, and that doesn't seem like a great idea.  The Helm project itself really seems like it wants to move on and forget about Helm 2.

See: https://v3.helm.sh/docs/faq/#changes-since-helm-2

Reply:
> I thought the v3 binary was totally compatible with v2 charts?

It does, but v2 of the charts is only for v3 of the binary, if I understand correctly.

> For what it's worth, among the people at KubeCon last year there were quite a few people that would have been quite happy to see Helm v2 burned to the ground, and pretty much everyone was glad to see Helm v3 get released. I don't know the technical details for all the angst with respect to v2, but I think the community has moved on.

I think that is totally true for new deployments, and I am one of those people who hates helm v2. If we don't have a good reason to support it, lets not! But I also know that customers have helmv2 out there, and might not be able to prioritize upgrading/reworking it.

Reply:
From some research - I see other projects going with just Helm v3 support, so I think we can merge this PR and then deal with requests for Helm v2 support down the line.